groups/sig-k8s-infra/groups.yaml

```diff
@@ -379,6 +379,18 @@ groups:
       - sig-k8s-infra-leads@kubernetes.io
       - steering@kubernetes.io
 
+  - email-id: k8s-infra-aws-accounting@kubernetes.io
+    name: k8s-infra-aws-accounting
+    description: |-
+      ACL for AWS accounting (readonly)
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - justinleegarrison@gmail.com
```

Comment:
verified for typos on justin's email:
https://cs.github.com/?scopeName=All+repos&scope=&q=justinleegarrison%40gmail.com